src/DotNetLightning.Core/Transactions/Transactions.fs

```diff
@@ -735,19 +736,29 @@ module Transactions =
                 else
                     spec.ToLocal.ToMoney(), spec.ToRemote.ToMoney() - closingFee
 
-            let maybeToLocalOutput = if (toLocalAmount >= dustLimit) then Some(toLocalAmount, localDestination) else None
-            let maybeToRemoteOutput = if (toRemoteAmount >= dustLimit) then Some(toRemoteAmount, remoteDestination) else None
-            let outputs = seq [maybeToLocalOutput; maybeToRemoteOutput] |> Seq.choose id
+            let maybeToLocalOutput = if (toLocalAmount >= dustLimit) then Some(TxOut(toLocalAmount, localDestination)) else None
+            let maybeToRemoteOutput = if (toRemoteAmount >= dustLimit) then Some(TxOut(toRemoteAmount, remoteDestination)) else None
```

Comment:
now that you're changing these 2 lines above, IMO it's better to split them, to not have so much logic in one line; like this (and removing unneeded parenthesis):

```
let maybeToLocalOutput =
    if toLocalAmount >= dustLimit then
        Some(TxOut(toLocalAmount, localDestination))
    else
        None
```